client/dive-common/components/ControlsContainer.vue

```diff
@@ -104,6 +104,10 @@ export default defineComponent({
       clientSettings.timelineCountSettings.defaultView = countView.value;
     };
 
+    function handleSelectTrack(trackId: number, modifiers: { ctrl: boolean } | null) {
+      emit('select-track', trackId, modifiers);
+    }
```

Comment:
Swap this from an emit to directly using `handler.trackSelect` from `const handler = useHandler()`.  I think that should work and we don't have to worry about emitting an event to Viewer.vue and handling it there.

Reply:
This is fixed by 84f2b78f